### ISSUE TITLE
LetsEncrypt - GANDI DNS - Deprecation of Certbot 3rd party plugins prefixes

### ISSUE THREAD

**Issue by GoetzGoerisch** [home-assistant/addons#3160]
### Describe the issue you are experiencing

The GANDI DNS configuration still uses the 3rd party plugin prefix, which is going to be removed.

Other DNS provides are already changed to the new format.

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### Which add-on are you reporting an issue with?

Let's Encrypt

### What is the version of the add-on?

4.12.9

### Steps to reproduce the issue

1. install addon
2. use GANDI and DNS Challenge
3. observe logs
...

## System Information

version | core-2023.7.3
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.11.4
os_name | Linux
os_version | 6.1.21-v8
arch | aarch64
timezone | Europe/Berlin
config_dir | /config

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 10.3
-- | --
update_channel | stable
supervisor_version | supervisor-2023.07.1
agent_version | 1.5.1
docker_version | 23.0.6
disk_total | 28.5 GB
disk_used | 9.6 GB
healthy | true
supported | true
board | rpi4-64
supervisor_api | ok
version_api | ok
installed_addons |  Let's Encrypt (4.12.9)

</details>

<details><summary>Dashboards</summary>

dashboards | 1
-- | --
resources | 0
mode | auto-gen

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | 27. Juli 2023 um 17:34
-- | --
current_recorder_run | 2. August 2023 um 16:17
estimated_db_size | 517.98 MiB
database_engine | sqlite
database_version | 3.41.2

</details>

### Anything in the add-on logs that might be useful for us?

```txt
[16:18:38] INFO: Selected DNS Provider: dns-gandi
[16:18:38] INFO: Use propagation seconds: 60
Saving debug log to /var/log/letsencrypt/letsencrypt.log
Plugin legacy name certbot-plugin-gandi:dns may be removed in a future version. Please use dns instead.
Certbot is moving to remove 3rd party plugins prefixes. Please use --authenticator dns-gandi --dns-gandi-credentials
```

### Additional information

Going to raise a PR to fix line:

https://github.com/home-assistant/addons/blob/a3ae943a923dad5d9961a8a9cd394f0c1920dc2b/letsencrypt/rootfs/etc/services.d/lets-encrypt/run#L95

**Comment by GoetzGoerisch** [comment]
Not stale.

**Comment by GoetzGoerisch** [comment]
Waiting for the PR to get a second review and merge.

**Comment by GoetzGoerisch** [comment]
Still waiting on a review of the PR. Not stale.

**Comment by GoetzGoerisch** [comment]
Fixed with 13e77331531d1228bbdba04faba4c583bf4d708c in #3325.
